### ISSUE TITLE
Cameras shown have already set correct camera

### ISSUE THREAD

**Issue by PiotrGackowski** [addshore/wikicrowd#55]
I got 
https://commons.wikimedia.org/wiki/File:Canon_EOS_M50_black_06.jpg
But inside there is already information about "Canon EOS M50". 

I remember that I got such situation couple times, but I closed browser. But I am really afraid that I am marking images that have already correct info.

**Comment by PiotrGackowski** [comment]
https://commons.wikimedia.org/wiki/File:Canon_EOS_M50_black_07.jpg

**Comment by waldyrious** [comment]
I didn't understand this issue at first, but I bumped into` the same problem, which clarified it: some edits I made with wikicrowd were reverted on Commons because the image already had a "depicts" statement with a Wikidata item that's a subclass/instance of camera.

For example, the file [File:Samsung NX2000 (White) top.jpg](https://commons.wikimedia.org/wiki/File:Samsung_NX2000_(White)_top.jpg) was shown to me in Wikicrowd and I clicked yes, which added a "depicts: [camera](https://www.wikidata.org/wiki/Q15328)" statement to it, but it already had a "depicts: [Samsung NX2000](https://www.wikidata.org/wiki/Q18354117)" statement.

**Comment by addshore** [comment]
So the question will not be generated, and also edit will not happen if the file in question has a depicts statement for any item in this list
https://query.wikidata.org/#SELECT%20DISTINCT%20%3Fi%20WHERE%7B%3Fi%20wdt%3AP31%2Fwdt%3AP279%2a%7Cwdt%3AP279%2Fwdt%3AP279%2a%20wd%3AQ15328%20%7D
or this list https://query.wikidata.org/#SELECT%20DISTINCT%20%3Fi%20WHERE%7Bwd%3AQ15328%20wdt%3AP279%2B%20%3Fi%20%7D
Basically looking for trees of items in the class tree from Q15328 as defined in https://github.com/addshore/wikicrowd/blob/main/spec/depicts/misc/camera.yaml#L1

This comes from this code https://github.com/addshore/wikicrowd/blob/main/app/Jobs/GenerateDepictsQuestions.php#L212-L238
trying to cover `instancesOfAndSubclassesOf` the top level item, and also `parentInstancesOfAndSubclassesOf`.

The model in question must not meet these criteria
https://www.wikidata.org/wiki/Q51539860
Going up the trees
 - instance of digital camera model (Q20741022)
   - subclass of camera model (Q20888659)
 - subclass of mirrorless interchangeable-lens camera (Q209918)
   - subclass of digital camera (Q62927)
     - subclass of camera (Q15328)

So going all the way up the subclass tree we can get to camera (Q15328)

So something is not right :)

**Comment by PiotrGackowski** [comment]
Still present, still problematic
https://commons.wikimedia.org/w/index.php?title=File:Samsung_NX2000_(White)_side.jpg&oldid=prev&diff=661035326

https://www.wikidata.org/wiki/Q18354117
it have "digital camera model"

**Comment by addshore** [comment]
So taking another look

- https://www.wikidata.org/wiki/Q18354117 Samsung NX2000 is instance of [digital camera model](https://www.wikidata.org/wiki/Q20741022)
- which is subclass of [camera model](https://www.wikidata.org/wiki/Q20888659)

And this is the top of the tree.
So the tool has no way of knowing that Q18354117 is a camera.

I believe this differs from Q51539860 I looked at above in https://github.com/addshore/wikicrowd/issues/55#issuecomment-1101782478 as that was also subclass of mirrorless interchangeable-lens camera (Q209918)

TLDR, the modelling on Wikidata or commons needs to change.
The current statement on https://commons.wikimedia.org/wiki/File:Samsung_NX2000_(White)_side.jpg does not say that the image depicts a camera, rather a model of camera, and a model of camera doesnt claim to be instance of camera.

Looking at the modeling of a 5D https://www.wikidata.org/wiki/Q64955
![image](https://user-images.githubusercontent.com/3308769/171812347-d5a0b5c9-d05e-4dea-a8c7-f159fd8aa5cc.png)

So the model should have a subclass of statement
So I went ahead and added it https://www.wikidata.org/w/index.php?title=Q18354117&diff=1653383331&oldid=1620502022

It shouldnt appear in questions anymore, as:
- [mirrorless interchangeable-lens camera](https://www.wikidata.org/wiki/Q209918)
- subclass of [digital camera](https://www.wikidata.org/wiki/Q62927)
- subclass of [camera](https://www.wikidata.org/wiki/Q15328)
